client/me/help/main.jsx

```diff
@@ -243,15 +241,16 @@ class Help extends React.PureComponent {
 	}
 }
 
-const isWPCOMBusinessPlan = purchase =>
-	planMatches( purchase.productSlug, { type: TYPE_BUSINESS, group: GROUP_WPCOM } );
+function purchaseIsWpComBusinessPlan( { productSlug } ) {
+	return isWpComBusinessPlan( productSlug );
+}
 
 export const mapStateToProps = ( state, ownProps ) => {
 	const isEmailVerified = isCurrentUserEmailVerified( state );
 	const userId = getCurrentUserId( state );
 	const purchases = getUserPurchases( state, userId );
 	const isLoading = isFetchingUserPurchases( state );
-	const isBusinessPlanUser = purchases && !! find( purchases, isWPCOMBusinessPlan );
+	const isBusinessPlanUser = some( purchases, purchaseIsWpComBusinessPlan );
```

Comment:
after this change is there much use in keeping the function external or should we inline it here?

Reply:
Inlining it would be good 👍